plugins/output/flight/flight_test.go

```diff
@@ -0,0 +1,44 @@
+package flight
+
+import (
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/stretchr/testify/require"
+)
+
+const servicePort = "9999"
+
+func TestConnectAndWriteIntegration(t *testing.T) {
+
+	var err error
+
+	if testing.Short() {
+		t.Skip("Skipping integration testing in short mode")
+	}
+
+	container := testutil.Container{
```

Comment:
Do we want to require Docker for running the integration tests? I think it is fine if the option for automatically creating and Apache Arrow Flight end point is available, but that it should be possible to execute the integration tests by manually creating an Apache Arrow Flight end point. What do you think @chrthomsen and @CGodiksen?

Reply:
I could not get the docker integration to work, as I don't think any implementation of Apache Arrow Flight exists as an official image. I have resorted to a different test strategy.